cmd/definitions/tests/.gitignore

```diff
@@ -0,0 +1 @@
+generated.go
```

Comment:
Why set `generated.go` ignored?

Reply:
`generated.go` is only used for testing and it's content is meanless, so that I think we can ignore it.